notebooks/en/fine_tuning_vit_custom_dataset.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>Maybe say something like the following to avoid confusion with the next sentence where you say we can see the features again.</p><p> </p><p>"We can the image is a <code>PIL.Image</code> with a label associated with it." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Sorry I didn't understand, where should I put this?